.github/workflows/e2e_libvirt.yaml

```diff
@@ -27,6 +27,8 @@ on:
 env:
   CLOUD_PROVIDER: libvirt
   DEBIAN_FRONTEND: noninteractive
+  KBS_REGISTRY: ghcr.io/confidential-containers/staged-images/kbs
+  KBS_CLIENT_REGISTRY: ghcr.io/confidential-containers/staged-images/kbs-client
```

Comment:
What are these envs being used for in this commit? Should they be part of a later one?

Reply:
nit: I saw in the next commit they were deleted, so they could be cleaned up in commit 1